integration/setup/setup.go

```diff
@@ -47,17 +49,32 @@ func installResources(ctx context.Context, config Config) error {
 		}
 	}
 
-	var operatorTarballPath string
+	var operatorTarballURL string
 	{
-		config.Logger.Debugf(ctx, "getting tarball URL")
+		config.Logger.Debugf(ctx, "getting %#q tarball URL", project.Name())
+
+		o := func() error {
+			operatorTarballURL, err = appcatalog.GetLatestChart(ctx, key.DefaultTestCatalogStorageURL(), project.Name(), env.CircleSHA())
+			if err != nil {
+				return microerror.Mask(err)
+			}
```

Comment:
Same as in app-operator we can't install chart-operator as an app. So we need a retry here.